frontend/src/lib/util/util.ts

```diff
@@ -42,18 +42,16 @@ export function getProjectRouteFromURL(url: URL) {
 
 function columnTypeOrder(colType: ZenoColumnType) {
 	switch (colType) {
-		case ZenoColumnType.FEATURE:
+		case ZenoColumnType.ID:
```

Comment:
Don't think this makes sense in many cases. Used this for putting feature columns up top in the selects.